platform_it/pom.xml

```diff
@@ -1043,6 +1043,7 @@
                                     -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=4020
                                     -Djava.security.krb5.conf=${project.build.directory}/krb5.conf
                                     -enableassertions
+                                    -disableassertions:com.yahoo.sketches.frequencies.ReversePurgeItemHashMap
```

Comment:
Should we disable these in the mem_sql pom file too?

Reply:
Actually, I don't know if it's needed. I didn't see this problem on mem platform. When looking into `mem_sql/pom.xml`, I didn't find `-enableassertions` arg sent to jvm. Do we generally have assertions enabled on mem platform?

Reply:
You're right, it looks like mem does not have assertions enabled.